.github/workflows/pr-check.yml

```diff
@@ -25,7 +74,21 @@ jobs:
 
       - run: yarn install --prefer-offline --frozen-lockfile
 
+  Backend:
+    needs: Setup
+    runs-on: ubuntu-latest
+    steps:
       - name: Test Backend
         working-directory: ./packages/backend
         run:
           yarn test
+
+  Frontend:
+    needs: Setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test Frontend
+        working-directory: ./packages/frontend
+        run:
+          yarn build
```

Comment:
Add a step for `yarn test` in the frontend

Reply:
cool, I will have a look